code/modules/mob/living/silicon/robot/robot.dm

```diff
@@ -4,6 +4,10 @@
 	spark_system.attach(src)
 
 	ADD_TRAIT(src, TRAIT_CAN_STRIP, INNATE_TRAIT)
+	AddComponent(/datum/component/tippable, \
+		tip_time = 2.5 SECONDS, \
```

Comment:
Should probably be a fair bit longer, TBH, as funny as it is

Reply:
yeah but then you have to flash them to tip them